src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -82,14 +82,9 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath) thro
         addressBookStorage.saveAddressBook(addressBook, filePath);
     }
 
-    @Override
-    public Path getModuleFilePath() {
-        return moduleMapStorage.getModuleFilePath();
-    }
-
     @Override
     public ModuleMap readModuleMap() {
-        logger.fine("Attempting to read data from file: " + moduleMapStorage.getModuleFilePath());
+        logger.fine("Attempting to read data from module map");
```

Comment:
Is there some way to get codecov to ignore this? I don't actually think there is a test in place to update here.

Reply:
I have no idea...